lib/bricks-evaluate.js

```diff
@@ -49,6 +49,24 @@ function createContext() {
 	extend(context, require('./bricks-net-api'));
 	
 	
+	// Add a `DemoObject` to demonstrate the JSON-POST.
```

Comment:
Nit: Please don't use "string" as the field name, since it's going to get C++ a bit crazy. Thanks!